src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -273,64 +271,56 @@ public void testMultipleSinkTaskWithLogs() throws Exception {
 
   @Test
   public void testSinkTaskWithMultipleOpenClose() throws Exception {
```

Comment:
Didn't write this test so I'm hoping I didn't reduce the test scope with my changes. Let me know if I should add/remove anything

Reply:
moved to a following pr, so this comment is out of context